src/cmd/common.go

```diff
@@ -131,5 +131,10 @@ func cliSetup(cmd *cobra.Command) error {
 			return fmt.Errorf(err.Error())
 		}
 	}
+
+	// TODO(schristoff): Leverage SDK and refactor
+	// This sets up zarf CLI with the same configuration options as UDSCLI
+	zarfCommon.SetupCLI(logLevel, config.SkipLogFile, config.NoColor)
```

Comment:
We moved `config.NoColor` to be a var and not put of the config, so this is the "way to pass it in", happy to move this function somewhere better suited if there is a place